docs/troubleshooting/reindex-elasticsearch.md

```diff
@@ -0,0 +1,29 @@
+---
+id: reindex-elasticsearch
+title: Reindexing Elasticsearch Guide
+hide_title: false
+---
+
+You may find yourself in a situation where you need to reindex your Elasticsearch index. If you have issues with your
+current index, or the mappings for an index have been updated (they do not update automatically), there are two ways
```

Comment:
Mention that a reindex due to mappings changing will be called out in the Migration Steps of the changelog.